hokusai/staging.yml

```diff
@@ -108,7 +108,7 @@ metadata:
   name: team-external-auth
   annotations:
     nginx.ingress.kubernetes.io/auth-url: "http://$host/api/auth"
-    nginx.ingress.kubernetes.io/auth-signin: "https://stagingapi.artsy.net/oauth2/authorize?client_id={{ APP_ID }}&redirect_uri=$host/oauth2/callback?redirect_to=$escaped_request_uri"
+    nginx.ingress.kubernetes.io/auth-signin: "http://$host/oauth2/callback?redirect_to=$escaped_request_uri"
```

Comment:
`$host` is the fully-qualified-domain name - see http://nginx.org/en/docs/http/ngx_http_core_module.html#var_host - I don't think it would work in terms of using one of kubernetes' internal DNS schemes (i.e. the service `team-web-internal.default.cluster.local`)